set.mm

```diff
@@ -465922,6 +466013,59 @@ its graph has a given second element (that is, function value).
       ZCKRUBCKSHLTJUA $.
   $}
 
+  ${
+    $d A f k l x y z $.  $d B f k l y z $.  $d C f x y z $.
+    $d ph f k l x y z $.
+    fsumiunle.1 $e |- ( ph -> A e. Fin ) $.
+    fsumiunle.2 $e |- ( ( ph /\ x e. A ) -> B e. Fin ) $.
+    fsumiunle.3 $e |- ( ( ( ph /\ x e. A ) /\ k e. B ) -> C e. RR ) $.
+    fsumiunle.4 $e |- ( ( ( ph /\ x e. A ) /\ k e. B ) -> 0 <_ C ) $.
+    $( Inequality for a sum over a non-disjoint indexed union.  (Contributed by
+       Thierry Arnoux, 1-Jan-2021.) $)
```

Comment:
Proposal, but it's as you prefer:
Upper bound for a sum of nonnegative terms over an indexed union. The inequality may be strict if the indexed union is non-disjoint, since in the right hand side, a summand may be counted several times.

Reply:
I like it! I'm doing the change.